tools/rapids-generate-version

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# A utility script that generates a package version
+set -exo pipefail
```

Comment:
```suggestion
set -euo pipefail
```

this aligns with the settings we use in other scripts.

Reply:
Done.